nodes/nodes-containers-downward-api.adoc

```diff
@@ -0,0 +1,41 @@
+:context: nodes-containers-downward-api
+[id='nodes-containers-downward-api_{context}']
+= Allowing containers to consume API objects
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+
+
+
+
+The _Downward API_ is a mechanism that allows containers to consume information about API objects without coupling to OpenShift Container Platform. Such information includes the pod’s name, namespace, and resource values. Containers can consume information from the downward API using environment variables or a volume plug-in.
+
+
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/nodes-containers-downward-api-about.adoc[leveloffset=+1]
+
+include::modules/nodes-containers-downward-api-container-values.adoc[leveloffset=+1]
+
+include::modules/nodes-containers-downward-api-container-values-envars.adoc[leveloffset=+2]
```

Comment:
Would you do both in the same flow?

Reply:
Probably not. They seem like different approaches.

Reply:
Then they should probably be part of different stories

Reply:
They are different way to use the Downward API to compete a task.

Reply:
Then I'd probably make them alternate steps in the same module or document only one. The assemblies are supposed to be single coherent flows. :man_shrugging: